proposals/lip-0074.md

```diff
@@ -920,6 +925,22 @@ def applyProposalOutcome(type: int, data: bytes) -> None:
         call updateConfig(dataObj.paramName, dataObj.value) method of module dataObj.moduleName
 ```
 
+#### validateConfigUpdate
+
+The function raises an exception if the proposed config update is invalid.
+
+```python
+def validateConfigUpdate(name: string, value: bytes):
+    type = type of value with name in configSchema
```

Comment:
To me this looks more readable:
```suggestion
    type = string indicating the type of name property in configSchema
```

Or maybe this?
```suggestion
    type = configSchema["properties"][name]["type"]
```

But probably it doesn't need to be changed, because then it would need to be changed in `updateConfig`, too... just a thinking-aloud-kind-of-comment, feel free to resolve.

Reply:
Agree, the current version is not very readable. I have changed to your first suggestion.